packages/flutter/test/cupertino/scaffold_test.dart

```diff
@@ -315,4 +315,55 @@ void main() {
     final BoxDecoration decoration = decoratedBox.decoration;
     expect(decoration.color, const Color(0xFF010203));
   });
+
+  testWidgets('Lists in CupertinoPageScaffold scroll to the top when status bar tapped', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        builder: (BuildContext context, Widget child) {
+          return MediaQuery(
+            data: MediaQuery.of(context).copyWith(padding: const EdgeInsets.only(top: 20)),
```

Comment:
Is this done to imitate a status bar?  If so, it would be great if we could codify that as something more than what looks like incidental padding.  Maybe a status bar widget just for testing?

Reply:
Added comment for now to avoid an indirection. Makes sense to make a function or something if there were any more status bar tests in the future.